yb-voyager/src/srcdb/pg_dump_export_data.go

```diff
@@ -39,8 +39,10 @@ func pgdumpExportDataOffline(ctx context.Context, source *Source, connectionUri
 	// Using pgdump for exporting data in directory format.
 	cmd := fmt.Sprintf(`pg_dump "%s" --no-blobs --data-only --no-owner --compress=0 %s -Fd --file %s --jobs %d --no-privileges --no-tablespaces`,
 		connectionUri, tableListPatterns, dataDirPath, source.NumConnections)
-
-	log.Infof("Running command: %s", cmd)
+	redactedUri := utils.GetRedactedURLs([]string{connectionUri})[0]
+	redactedCmd := fmt.Sprintf(`pg_dump "%s" --no-blobs --data-only --no-owner --compress=0 %s -Fd --file %s --jobs %d --no-privileges --no-tablespaces`,
```

Comment:
Each time you change the main `cmd`, you will have to change the `redactedCmd`. What if we do something like:
```
pgDumpArgs := fmt.Sprintf("--no-blobs --data-only --no-owner --compress=0 %s -Fd --file %s --jobs %d --no-privileges --no-tablespaces", ...)
cmd := fmt.Sprintf(`pg_dump "%s" %s`, connectionUri, pgDumpArgs)
redactedCmd := fmt.Sprintf(`pg_dump "%s" %s`, redactedUri, pgDumpArgs)
```